### PR TITLE
Neighborhood & web links

### PR DESCRIPTION
**Pull request by fayeed**

### CODE REVIEW

**Review by jdeepee — COMMENTED** [review]
Not sure of feature scope for this PR, but I tried a few https & neighbourhood links to try things out. I couldn't get any link cards showing for https links, tried posting youtube, soundlcloud and twitter links. Neighbourhood links loaded great, clicking on already joined communities correctly navigated to that community but on communities I hadn't joined, clicking caused no action.

**Comment by fayeed** [comment]
@jdeepee I removed the URL cards but neighborhood cards should work. I will look into the card opening the modal.

File: packages/junto-utils/api/hideEmbeds.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+export default async function ({
+  perspectiveUuid,
+  messageUrl,
+}: Payload) {
+  try {
+    await ad4mClient.perspective.addLink(
+      perspectiveUuid,
+      new Link({
+        source: messageUrl,
+        target: "-",
```
[the remark this answers is not in the document]
Comment (fayeed) [reply packages/junto-utils/api/hideEmbeds.ts:19]: Omitting the target throws an error I will put in `flux://null`

File: packages/junto-utils/api/getNeighbourhoodLink.ts [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+    const hoods = []
+
+    if (!isHidden) {
+      for (const neighbourhood of neighbourhoods) {
+        const exp = await ad4mClient.expression.get(neighbourhood);
+        const links = JSON.parse(exp.data).meta.links;
+
+        const languageLinks = links.filter(findLink.language);
+        const langs = await getMetaFromLinks(languageLinks);
```
[the remark this answers is not in the document]
Comment (fayeed) [reply packages/junto-utils/api/getNeighbourhoodLink.ts:57]: yeah likely there were a lot of merge conflicts so might have missed on this.

**Pull request merged**